gcp-deployer/README.md

```diff
@@ -104,3 +104,24 @@ The yaml is unmarshalled into a [machinesetup.configList](../cloud/google/machin
 Right now, the startup scripts must be bash scripts. 
 There is a set of environment variables that is concatenated to the beginning of the startup script.
 You can find them [here](../cloud/google/metadata.go) and use them in your startup script as needed.
+
+## Contributing
+
+### Building
+To build a binary, run the following command from this folder.
+
+```
+go build .
+```
+
+### Testing
+To run the unit tests, run the following command from the parent of this folder.
+
+```
+go test ./...
```

Comment:
Yea, I agree developers should run the unit tests from the entire project, especially since they should run pretty quickly. I'd recommend adding a comment explaining what will run, since the integration test section is about this specific tool but the unit test section is about the entire project. Also, this tool doesn't actually have any unit tests, so it could be a little confusing.